pkg/storaged/anaconda.jsx

```diff
@@ -22,19 +22,41 @@ import client from "./client.js";
 import { decode_filename } from "./utils.js";
 import { parse_subvol_from_options } from "./btrfs/utils.jsx";
 
+function parse_parent_from_options(options) {
+    const parent_match = options.match(/x-parent=(?<parent>[\w\\-]+)/);
+    if (parent_match) {
+        return parent_match.groups.parent;
+    } else
+        return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19951-20240209-081459-7aeeff60-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)